anesthetic/samples.py

```diff
@@ -319,33 +230,58 @@ def plot_2d(self, axes, *args, **kwargs):
             Pandas array of axes objects
 
         """
-        default_types = {'diagonal': 'kde', 'lower': 'kde', 'upper': 'scatter'}
-        types = kwargs.pop('types', default_types)
+        kind = kwargs.pop('kind', 'default')
+        kind = kwargs.pop('kinds', kind)
+        try:
+            kind = self.plot_2d_default_kinds.get(kind, kind)
+        except TypeError:
+            pass
```

Comment:
This doesn't fully work, make failing test and fix.
Maybe simply check for a string and use the dictionary.

Reply:
Done.